setup.cfg

```diff
@@ -104,6 +104,9 @@ openai =
     openai~=0.27.8
     tiktoken~=0.3.3
 
+lit-gpt =
+    lit-gpt @ git+https://github.com/Lightning-AI/lit-gpt@main
```

Comment:
I _think_ PyPI will reject packages with dependencies from GitHub, but we can deal with this in a later PR.

Reply:
true, no dependency outside pypi like source is accepted, you need to drop them